tests/catalog-schema.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
I need help with what this file is for/the file name:

I think a tap can have a `catalog.json`.
The catalog can have multiple streams, and each stream can have a schema.

...Is there a catalog schema too?

Reply:
Ah, I think I get it: take the catalog, and use jsonschema to get the structure of it. ...That's what's in this file...

👍

Reply:
Yes :+1: